Xamarin.Forms.Core.UnitTests/ResourceDictionaryTests.cs

```diff
@@ -253,4 +253,13 @@ public void MultiLevelMerge ()
 			Assert.Fail ();
 		}
 	}
+
```

Comment:
Please add the unit test inside the test class. It won't compile otherwise. :) 👎 until fixed (to prevent merge)